src/pages/home/report/ReportActionCompose.js

```diff
@@ -397,8 +408,6 @@ class ReportActionCompose extends React.Component {
                         onClose={this.hideEmojiPicker}
                         onModalShow={this.focusEmojiSearchInput}
                         hideModalContentWhileAnimating
-                        animationInTiming={1}
```

Comment:
Why are we removing this? There's a comment above saying why they're there.

Reply:
So It was added to disable the animation  as the author found it laggy but I was requested to enable animation on emoji picker thus I removed it. I will remove the comments as well.